src/invidious/views/data_control.ecr

```diff
@@ -8,14 +8,14 @@
             <legend><%= translate(locale, "Import") %></legend>
 
             <div class="pure-control-group">
-                <label for="import_youtube"><%= translate(locale, "Import Invidious data") %></label>
+                <label for="import_youtube"><%= translate(locale, "Import Invidious JSON data") %></label>
```

Comment:
I don't know what I need to do for the locale files, could someone explain?

Reply:
The string that you pass to `translate()` is the identifier of the string. This is then looked up in the specified `locale/<locale>.json` to give a translation.

E.g, if you're doing:
```crystal
translate("fr", "Import YouTube subscriptions")
```
You'll get the following :
```crystal
"Importer des abonnements YouTube"
```

https://github.com/iv-org/invidious/blob/e2fc64296dacf6ce8a406183c1bb6388dce6d117/locales/fr.json#L35

Reply:
Yeah, you only need to update the json file. As for the identifier, I'll take care of that in the follow-up PR of #2871, where I'll rework the user interface.